src/gui/addnewtorrentdialog.ui

```diff
@@ -415,6 +415,9 @@
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
+           <property name="sizeType">
+            <enum>QSizePolicy::Maximum</enum>
+           </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>168</width>
```

Comment:
Nitpick, what about rounding it to `170` or `165`? It is a bit unnerving (for me) to see a arbitrary number.

Reply:
Will do.